msal_extensions/cache_lock.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, lockfile_path):
         self._lockpath = lockfile_path
         # Support for passing through arguments to the open syscall was added in v1.4.0
         open_kwargs = ({'buffering': 0}
-            if LooseVersion(portalocker.__version__) >= LooseVersion("1.4.0") else {})
+            if Version(portalocker.__version__) >= Version("1.4.0") else {})
```

Comment:
I don't think `LooseVersion` is really needed here, so `Version` should be good enough.

Reply:
The `LooseVersion()` was introduced by @bluca . What do you think with this change, Luca?

Reply:
looks good to me, thanks